modules/ROOT/pages/first-look/linux-first-look.adoc

```diff
@@ -0,0 +1,140 @@
+= Linux first look guide
+
+== Prerequisites
+ * Java 17
+ * A Neo4j cluster meeting *xref:addition/instance-requirements.adoc[these]* requirements
+
+NOTE: The query log feature is only available for DBMSs with instances that are running Neo4j 5.1.0 or higher.
+
+== Create persistence DBMS
+
+=== Download and extract
+Download latest Neo4j Linux/Mac executable from https://neo4j.com/download-center/#ops-manager[here] and extract the package. 
+Extracted directory location will be referred to as NEO4J_HOME form now on.
+
+=== Change password
+Navigate to NEO4j_HOME and run:
+
+[source, terminal, role=noheader]
+----
+./bin/neo4j-admin dbms set-initial-password lovelypassword
+----
+
+=== Run Neo4j 
+[source, terminal, role=noheader]
+----
+./bin/neo4j start
+----
+
+See link:https://neo4j.com/docs/operations-manual/current/installation/linux/tarball/[Neo4j Linux installation documentation] for more details about installing and running Neo4j.
+
+== Server installation
+=== Unpack NOM server bundled with Neo4j
+   
+[source, terminal, role=noheader]
+----
+   tar -xzf NEO4J_HOME/products/neo4j-ops-manager-server-*-unix.tar.gz
+----
+    
+Extracted directory will now be known as NOM_SERVER_HOME.
+
+=== Generate self-signed certificates
+
+Navigate to NOM_SERVER_HOME and run:
+
+[source, terminal, role=noheader]
+----
+java -jar ./lib/server.jar ssc -n localhost \
+        -o ./certificates \
+        -p secretpassword \
```

Comment:
Can we make the secretpassword stuff stand out? Maybe in the text before? Or are we okay with them generating certs with that password? I know this is only for eval/demo use but maybe it would be wise to not have a hard-coded secret in case people actually wrongly use it in production

Reply:
Good point - I think we can make it bold and wrap it in angled brackets. AND make a note that it should be changed. Because you are right - someone will use this in production...